test/login.e2e-spec.ts

```diff
@@ -0,0 +1,75 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication, HttpStatus } from '@nestjs/common';
```

Comment:
I suggest ordering packages and keywords:
```suggestion
import { HttpStatus, INestApplication, } from '@nestjs/common';
import { Test, TestingModule } from '@nestjs/testing';
```

Reply:
Resolved on d276e8230d000254a8224fae1632748e1c101d7d.